From 47d0b2bf45f2ce363e63f83f914b64218f35748b Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 28 Apr 2024 13:22:39 +0200
Subject: [PATCH 34/N] Revert "makepkg: bash v4 compat"

This reverts commit 1fbb46db545fad72daab79d769f19a4ae4cea6e3.

We are on bash v5 for a long time now

See #29
---
 scripts/makepkg.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/makepkg.sh.in b/scripts/makepkg.sh.in
index a5728fa..1585982 100644
--- a/scripts/makepkg.sh.in
+++ b/scripts/makepkg.sh.in
@@ -424,7 +424,7 @@ run_function() {
 
 		$pkgfunc &>"$logpipe"
 
-		wait $teepid
+		wait -f $teepid
 		rm "$logpipe"
 	else
 		"$pkgfunc"
